src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
 body {
   margin: 0;
+  width: 1024px;
+  height: 1044px;
```

Comment:
Similar to the width, setting a fixed height for the body can lead to problems on different screen sizes. It's better to use a flexible approach to accommodate various content lengths and screen dimensions.